script/changelog/api.ts

```diff
@@ -1,60 +1,126 @@
-import * as HTTPS from 'https'
+const octokit = require('@octokit/rest')({
+  timeout: 0,
+  requestMedia: 'application/vnd.github.v3+json',
+  headers: {
+    'User-Agent': 'what-the-changelog',
+  },
+})
 
-export interface IAPIPR {
+octokit.authenticate({
+  type: 'token',
+  token: process.env.GITHUB_ACCESS_TOKEN,
+})
+
+export interface IDesktopPullRequest {
   readonly title: string
   readonly body: string
+  readonly collaborators: ReadonlyArray<string>
+  readonly commits: ReadonlyArray<string>
 }
 
-type GraphQLResponse = {
-  readonly data: {
-    readonly repository: {
-      readonly pullRequest: IAPIPR
-    }
-  }
+interface IAPIUser {
+  readonly login: string
 }
 
-export function fetchPR(id: number): Promise<IAPIPR | null> {
-  return new Promise((resolve, reject) => {
-    const options: HTTPS.RequestOptions = {
-      host: 'api.github.com',
-      protocol: 'https:',
-      path: '/graphql',
-      method: 'POST',
-      headers: {
-        Authorization: `bearer ${process.env.GITHUB_ACCESS_TOKEN}`,
-        'User-Agent': 'what-the-changelog',
-      },
-    }
+interface IAPIPullRequest {
+  readonly title: string
+  readonly body: string
+  readonly user: IAPIUser
+}
 
-    const request = HTTPS.request(options, response => {
-      let received = ''
-      response.on('data', chunk => {
-        received += chunk
-      })
-
-      response.on('end', () => {
-        try {
-          const json: GraphQLResponse = JSON.parse(received)
-          const pr = json.data.repository.pullRequest
-          resolve(pr)
-        } catch (e) {
-          resolve(null)
-        }
-      })
+interface IAPICommit {
+  readonly sha: string
+  readonly author?: IAPIUser
+  readonly committer?: IAPIUser
+}
+
+interface IAPITeam {
+  readonly name: string
+  readonly id: number
+}
+
+interface IAPITeamMember {
+  readonly login: string
+}
+
+const webflowAccount = ['web-flow']
```

Comment:
can we put in a comment to describe what 'web-flow' / why it's a member of the core team?